domains.txt

```diff
@@ -468,13 +521,16 @@ nymanresourcing.com
 oakwellhampton.com
 occ-computing.co.uk
 octopus.net.uk
+officerecruitment.co.uk
 oho.co.uk
 oliverbernard.co.uk
+omniarc.co.uk
```

Comment:
Omniarc was established to provide a Recruitment and IT Services to SME's. Founded in 2005 we realised that there was a lack of specialised Recruitment and IT consultancy that dealt exclusively with the growing pains of an SME.